db/migrate/20180418140353_add_input_to_ticket_create.rb

```diff
@@ -0,0 +1,47 @@
+class AddInputToTicketCreate < ActiveRecord::Migration[5.1]
+  def up
+    # return if it's a new setup
+    return if !Setting.find_by(name: 'system_init_done')
+
+    ObjectManager::Attribute.add(
+        force: true,
+        object: 'Ticket',
+        name: 'organization_id',
+        display: 'Organization',
+        data_type: 'select_organization',
+        data_option: {
+            relation: 'Organization',
+            nulloption: false,
+            multiple: false,
+            null: false,
+            default: 0,
+            translate: true
+        },
+        editable: false,
+        active: true,
+
+        screens: {
+          create_middle_org: {
```

Comment:
See first comment.

Reply:
Uniq screen for new field as 'create_middle', 'create_top' and etc.

Reply:
If I would use generic hooks then on ticket create form there appear wrong additional fields with the same hooks.

Reply:
Then we need to change the `priority` of those so that the order matches the expectations.